### ISSUE TITLE
Location on disc of Mapbox GL Offline Packs

### ISSUE THREAD

**Issue by asterixorobelix** [mapbox/mapbox-gl-native#13660]
I cant find where Mapbox GL saves Offline Packs, since I need to copy the pack to an Android device in order to merge with the database.

### Steps to reproduce

 1. Download Mapbox.GL.app.zip macos-v0.13.0 as described here (https://www.mapbox.com/android-docs/maps/overview/offline/#offline-sideloading)
 2. Unzip and open
 3. Go to Window ‣ Offline Packs. Click the + button to create a new offline package. Enter a name for this offline package, then click Add. 
4. Unable to find where the newly created offline pack file has been saved on disc. I need to include this file on the android device in order to merge the offline pack with the main database.

### Expected behavior

I would expect that I can locate the downloaded offline pack on disc somehow so that I can copy the file over to an android device via USB.

### Actual behavior

I am unable to see where the offline pack is saved and copy it.

### Configuration

**Mapbox SDK versions:** 
macos-v0.13.0
**iOS/macOS versions:** 
MacOS Mojave (10.14.10
**Device/simulator models:**
N/A
**Xcode version:** 
N/A

**Comment by asterixorobelix** [comment]
Hi Tobrun, or if there was a way in the UI to change the location which the
offline packs save to?

Regards,
Nathan Stasin

**Comment by asheemmamoowala** [comment]
The iOS version of this [offline sideloading doc](https://www.mapbox.com/ios-sdk/maps/overview/offline/#generate-an-offline-package) includes the correct location. 

> Once an offline package finishes downloading, it will be completely contained in the offline database file at the path: `~/Library/Application\ Support/com.mapbox.MapboxGL/.mapbox/cache.db`

@asterixorobelix there is currently no way to change the location where offline packs are saved to in the MacOS app. The [command line tool](https://www.mapbox.com/android-docs/maps/overview/offline/#use-the-command-line-tool) offers more flexibility in creating offline packages.

**Comment by asterixorobelix** [comment]
Hi @asheemmamoowala 

But that directory (~/Library/Application\ Support/com.mapbox.MapboxGL/.mapbox/) does not seem to exist on my mac. See attached screenshots. It seems to me as if the offline packs are being saved elsewhere on the mac

<img width="771" alt="searching_ this_mac _and_re___mapbox_mapbox-gl-native__location_on_disc_of_mapbox_gl_offline_packs___13660
<img width="762" alt="searching_ this_mac" src="https://user-images.githubusercontent.com/20531795/50607734-db7f8800-0ed2-11e9-9508-5d66b76f4cab.png">
" src="https://user-images.githubusercontent.com/20531795/50607696-ca367b80-0ed2-11e9-9b1d-de9132bb8699.png">

Also, I have tried the Command line tool, but I get the following error:

/bin/sh: cmake: command not found
make: *** [build/macos/mbgl.xcodeproj] Error 127

**Comment by asheemmamoowala** [comment]
I think files/folders beginning with `.` are hidden in Finder. Try navigating there using the Terminal app

> Also, I have tried the Command line tool, but I get the following error:

>/bin/sh: cmake: command not found
>make: *** [build/macos/mbgl.xcodeproj] Error 127

Please follow the [Install](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/macos/INSTALL.md) instructions to setup your environment to build the SDK tools

**Comment by asterixorobelix** [comment]
@asheemmamoowala thanks for the input.

Presumably, the Mapbox GL.app GUI is intended to make things easier for people to save offline packs or for people who are less technically minded. However, not being able to easily find the save location of the offline pack means that this goal is not achieved. I also did searches for "Library" and "Application Support" and couldnt find the offline packs from those either. Even a button in the GUI which opens the save location for the offline pack would be a help.

I have built the command line tool and will use that
